pom.xml

```diff
@@ -2,12 +2,12 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
 
     <modelVersion>4.0.0</modelVersion>
-    <groupId>io.ecocode</groupId>
-    <artifactId>ios-plugin</artifactId>
-    <version>1.1.1-SNAPSHOT</version>
+    <groupId>org.green-code-initiative</groupId>
+    <artifactId>creedengo-ios-plugin</artifactId>
+    <version>2.0.0-SNAPSHOT</version>
     <packaging>pom</packaging>
 
-    <name>ecoCode iOS plugin</name>
+    <name>Creedengo iOS plugin</name>
     <description>Help the earth, adopt this green plugin for your applications</description>
     <inceptionYear>2023</inceptionYear>
```

Comment:
maybe put this "inceptionYear" to 2024 ? and run maven licence process

Reply:
done !